### ISSUE TITLE
Ignoring files/directories

### ISSUE THREAD

**Issue by bstrahija** [nfriedly/node-bestzip#14]
Is there a way to ignore some files/directories?

**Comment by nfriedly** [comment]
No, not at the moment. The features are limited to what's available in native zip commands.

**Comment by bstrahija** [comment]
The native zip command offers -x where you can do this. Thats why I ask

**Comment by nfriedly** [comment]
Ok, cool. I don't think I'll able to add it very soon, but go ahead and send a pull request if you can.
